CONP-PCNO/conp-dataset
# Multimodal dataset addition

## Description
This PR deals with adding a an FRDR dataset submodule to conp-dataset

## HOW TO TEST:
1) install conp-dataset
```datalad install -r <conp-datasets>``` && ```cd projects/Multimodal-data-with-GCaMP-imaging-sub-cortical-recording```

3) make a virtual environment 

2) launch configuration file
```./config.sh```

4) download a dataset file
```datalad get path/to/file```


Mandatory files and elements:
- [ ] A `README.md` file, at the root of the dataset
- [ ] A `DATS.json` file, at the root of the dataset
- [ ] If configuration is required (for instance to enable a special remote), a `config.sh` script at the root of the dataset
- [ ] A DOI (see instructions in [contribution guide](https://github.com/CONP-PCNO/conp-dataset/blob/master/.github/CONTRIBUTING.md), and corresponding badge in `README.md`

Functional checks:
- [ ] Dataset can be installed using DataLad, recursively if it has sub-datasets
- [ ] Every data file has a URL
- [ ] Every data file can be retrieved or requires authentication
- [ ] `DATS.json` is a valid DATs model
- [ ] If dataset is derived data, raw data is a sub-dataset


## Related issues 
We found an issue with the ``fair-research`` module having a ``__init__.py`` in the tests directory. This is currently causing our tests to fail. This is the PR solving the problem: https://github.com/fair-research/native-login/issues/46
We are waiting for it to be approved
